src/Engine/Component/AnimationController.hpp

```diff
@@ -24,6 +24,34 @@ namespace Component {
              */
             ENGINE_API void UpdateAnimation(float deltaTime);
 
+            /// Set a bool in the state machine.
+            /**
+             * @name Name of the bool.
```

Comment:
Parameters are documented as `@param parameterName Description`. Applies to the upcoming four methods.